### ISSUE TITLE
Sorting project

### ISSUE THREAD

**Issue by baldurpan** [DockStation/dockstation#104]
A minor feature request.
It would be super nice to be able to change the order of the projects.
Looks like it's currently sorted by time of adding?

**Comment by igor-lemon** [comment]
Hi @baldurpan. 
Yep.
Ok, accepted.

**Comment by adnoh** [comment]
That would be so nice. I have a bunch of projects and I don't find anything without the searchbar

**Comment by igor-lemon** [comment]
@adnoh But we have a search bar.
![image](https://user-images.githubusercontent.com/14764799/82221314-7fcb6880-9928-11ea-9ca7-fb4f613220c9.png)

**Comment by adnoh** [comment]
@igor-lemon yes, this is why i wrote i don't find anything without (using) the searchbar...and i don't always remember the name of my projects. but ordering them by name or creation or modification date would help me a lot. maybe also thing about manual reordering or grouping projects with a tag or folder (tags would be great)

**Comment by BalintCsuthy** [comment]
The projects order can be changed with changing/reordering the projects property in ~/.config/dockstation/Settings json file. Tested on ubuntu.
Not nice, but at least possible.
